releasenotes/notes/drop-gateway-alpha.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-managment
+releaseNotes:
+- |
+  **Removed** `alpha` Gateway API types by default. These can be explicitly re-enabled with `PILOT_ENABLE_ALPHA_GATEWAY_API=true`.
```

Comment:
I would call out `ReferenceGrant` explicitly as something that would be disabled here since it's in the stable channel of Gateway API. Thinking about it more, isn't this technically a breaking change?

Reply:
hmm, ReferenceGrant went beta it looks like... so I think we should actually switch to reading the beta version then do this.

And yes it is a breaking change but its for alpha APIs so I think its ok?

Reply:
Oh it sure did; the order was reversed in the CRD I was reading. In that case yes, we should move to that version first then do this